### DISCUSSION TITLE
Meta: Deprecation of team discussions

### DISCUSSION THREAD

**Discussion by jsamoocha** [hozn/stravalib#343] [Team Posts]
See the banner ☝️ from Github. Should we enable organization discussions as the alternative? I guess @lwasser may have permission to do that?

**Comment by lwasser** [comment]
oh - wow i missed that. great catch! yes lets

1. make sure that you have permission to enable discussions too!  i'll make you an org owner @jsamoocha 
2. then enable them and chat there!!

**Reply by lwasser** [reply]
magic!! @jsamoocha you now have super powers in this org 🎉 !! you are an owner ✨ 
And look - i've moved all of our team discussions here. let me know if you see any other issues with discussions etc that need to be addressed!!
